test-harness/scenarios/exceptions.circular.reference.oas3.scenario

```diff
@@ -75,18 +75,13 @@ extends: spectral:oas
 OpenAPI 3.x detected
 
 {document}
-  1:1   warning  oas3-api-servers                   OpenAPI `servers` must be present and non-empty array.
-  1:1   warning  openapi-tags                       OpenAPI object should have non-empty `tags` array.
-  10:9  warning  operation-operationId              Operation should have an `operationId`.
-  10:9  warning  operation-tags                     Operation should have non-empty `tags` array.
- 20:17    error  oas3-valid-oas-content-example     can't resolve reference #/components/schemas/links from id #
```

Comment:
@P0lip This PR is titled `fix: handle missing references more gracefully`, however this change in the test makes me a bit uneasy. 

We didn't fix anything and made the test pass. We've rather silently "swallowed" the fact we can't handle this....

...and I can't help by feel uneasy with it.

Reply:
> We've rather silently "swallowed" the fact we can't handle this....

Yeah, we did exactly this.
The title might be indeed a bit unfortunate.
Any ideas?

Reply:
>  So our own resolver isn't "bug free". 

It's certainly not.
The reason I'm swallowing the errors is that it's something user has no control of. Basically, it's not the $ref that is broken, but our resolver.
Hopefully we switch to a different resolver soon.

> How about making this tile a bit more upfront with something like `hack: swallow ajv missing reference". I understand this might be seen as a tad "provocative", but it might make future git-blames or GH search a bit easier would we need to.


Makes perfect sense.